### PR TITLE
[FEATURE] Add Site Settings to Docs

### PR DESCRIPTION
**Pull request by susannemoog**
See Core Feature 91080
See #841

### CODE REVIEW

File: Documentation/ApiOverview/SiteHandling/SiteSettings.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Add a `settings` block to the :file:`config.yml`:
+
+.. code-block:: yaml
+
+   settings:
+      categoryPid: 658
+      styles:
+         content:
+            loginform:
+               pid: 23
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/SiteSettings.rst:28]: Maybe add a comment that this fills a [predefined constant of EXT:fe_login](https://github.com/TYPO3/TYPO3.CMS/blob/10.4/typo3/sysext/felogin/Configuration/TypoScript/constants.typoscript#L10)?

Comment (susannemoog) [reply Documentation/ApiOverview/SiteHandling/SiteSettings.rst:28]: added a note on that.

File: Documentation/ApiOverview/SiteHandling/SiteSettings.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+.. note::
+
+   This example shows how to fill two constants of EXT:felogin via site settings.
```
Comment (mbrodala) [inline Documentation/ApiOverview/SiteHandling/SiteSettings.rst:32]: Actually _one_ constant, `categoryPid` is custom after all ...

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ApiOverview/SiteHandling/SiteSettings.rst (hunk 2 above)
Comment (susannemoog) [reply Documentation/ApiOverview/SiteHandling/SiteSettings.rst]: sometimes... I annoy myself - fixed, thanks!

**Pull request merged**